lib/src/requests.dart

```diff
@@ -51,7 +51,7 @@ class Response {
     return utf8.decode(bytes(), allowMalformed: true);
   }
 
-  dynamic json() {
+  dynamic? json() {
```

Comment:
Why is nullable dynamic needed?

Reply:
Looks like it is not needed. I'm not sure why I did this way.

Anyway I fixed it.